openfisca_france/parameters/prestations/transport/aide_financement_permis_apprenti.yaml

```
@@ -0,0 +1,7 @@
+reference: https://www.legifrance.gouv.fr/eli/decret/2019/1/3/MTRD1835610D/jo/texte
+age_minimal:
```

J'ai l'impression que les suffixes `max` et `min` sont [les plus utilisés](https://fr.openfisca.org/legislation/?q=max#search-input) dans le modèle actuel.

```suggestion
age_min:
```